tests/it/fix.rs

```diff
@@ -56,7 +56,6 @@ fn fix_output_filters() -> Vec<(&'static str, &'static str)> {
 #[tokio::test]
 async fn with_successful_http_no_auth_integration() {
     guard_integration_test!();
-
```

Comment:
Why remove the separating lines in these files? Was this automatic by `rustfmt`?

Generally I think separating blank space is useful for grouping related parts of a function together; it keeps it from reading like one big mush of commands.

Reply:
Good catch! I was removing my print statements used for debugging, which resulted in me accidentally removing the blank spaces.